package.json

```diff
@@ -91,7 +91,7 @@
     "lint": "eslint js test bench",
     "start-docs": "npm run build-min && npm run build-docs && jekyll serve -w",
     "start": "npm run download-bench-data && node server.js",
-    "test-suite": "node test/render.test.js && node test/query.test.js # update ci.sh if test invocation changes",
-    "test": "npm run lint && npm run build-min && tap --reporter dot test/js/*/*.js # update ci.sh if test invocation changes"
+    "test-suite": "node test/render.test.js && node test/query.test.js # && rem update ci.sh if test invocation changes",
+    "test": "npm run lint && npm run build-min && tap --reporter dot test/js/*/*.js # && rem update ci.sh if test invocation changes"
```

Comment:
I would prefer remove these comments rathern than complicate this file with cross-platform shell hacks.